app/src/main/java/fr/free/nrw/commons/notification/NotificationActivity.java

```diff
@@ -87,6 +89,9 @@ public boolean onSupportNavigateUp() {
 
     @SuppressLint("CheckResult")
     public void removeNotification(Notification notification) {
+        if(isRead) {
```

Comment:
Style: One space after `if` like in the similar lines you added as part of this commit.